{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T10:20:00.000+0000",
      "latitude": 50.728729093824036,
      "estimatedRestoration": "2022-05-13T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT1044'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT1044",
      "reference": "JT1044",
      "name": "BH5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH5 2AG",
        "BH5 2AP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.8283555898681083,
          50.728729093824036
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-05-13T10:00:00.000+0000",
      "UUID": "a7eaa298-5e77-4a33-9dd3-481542362ffd",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8283555898681083,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T09:14:00.000+0000",
      "latitude": 51.59798251064684,
      "estimatedRestoration": "2022-05-13T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT1010'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT1010",
      "reference": "JT1010",
      "engineerOnSiteTime": "2022-05-13T09:30:00.000+0000",
      "name": "OX10 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX10 0DG",
        "OX10 0XE"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.1266854825685328,
          51.59798251064684
        ],
        "type": "Point"
      },
      "UUID": "4cc99e3a-46b7-409c-acf8-7275467e1cad",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1266854825685328,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T11:19:00.000+0000",
      "latitude": 51.55597609946303,
      "estimatedRestoration": "2022-05-13T14:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT1150'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT1150",
      "reference": "JT1150",
      "name": "SN5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN5 6EG",
        "SN5 6EH",
        "SN5 6EL",
        "SN5 6EQ",
        "SN5 6HP",
        "SN5 6HS",
        "SN5 6HT",
        "SN5 6HU",
        "SN5 6HW",
        "SN5 6HX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.846900623051116,
          51.55597609946303
        ],
        "type": "Point"
      },
      "UUID": "bbd2649b-7a08-460a-847c-45495745593c",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.846900623051116,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T12:00:00.000+0000",
      "latitude": 51.70210456765957,
      "estimatedRestoration": "2022-05-13T15:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '207773'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/207773",
      "reference": "207773",
      "name": "GL7 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GL7 4BT"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.7901628801359695,
          51.70210456765957
        ],
        "type": "Point"
      },
      "UUID": "798096b4-23d0-4b96-96b9-c3f1788f7aa2",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7901628801359695,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T13:18:00.000+0000",
      "latitude": 51.58269925367015,
      "estimatedRestoration": "2022-05-13T16:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT1217'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT1217",
      "reference": "JT1217",
      "name": "SN2 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN2 1QN",
        "SN2 1QQ",
        "SN2 1QW",
        "SN25 3BG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.7844201215861366,
          51.58269925367015
        ],
        "type": "Point"
      },
      "UUID": "9aae0f1a-78bc-4b52-96cb-ef6bed80ee72",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7844201215861366,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T13:35:00.000+0000",
      "latitude": 50.831723282937745,
      "estimatedRestoration": "2022-05-13T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT1225'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT1225",
      "reference": "JT1225",
      "name": "PO19 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO19 8DN",
        "PO19 8FX",
        "PO19 8NS",
        "PO19 8NX",
        "PO19 8PD"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.7666060085461581,
          50.831723282937745
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-05-13T14:30:00.000+0000",
      "UUID": "aeebf666-290c-47a9-874f-7de7abfefd65",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7666060085461581,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T09:41:00.000+0000",
      "latitude": 56.275696451460405,
      "estimatedRestoration": "2022-05-13T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT1063'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT1063",
      "reference": "JT1063",
      "name": "PA32 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PA32 8XH"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -5.024128475385495,
          56.275696451460405
        ],
        "type": "Point"
      },
      "UUID": "65521210-caf7-4ee9-b1ef-1426b2e6b06f",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.024128475385495,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T07:32:00.000+0000",
      "latitude": 57.53973224155084,
      "estimatedRestoration": "2022-05-13T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT0986'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT0986",
      "reference": "JT0986",
      "name": "AB53 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "AB53 4DF",
        "AB53 4GB",
        "AB53 4GD",
        "AB53 4LG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.44879546104892,
          57.53973224155084
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-05-13T09:00:00.000+0000",
      "UUID": "d2edc831-4d1c-4e84-be67-e7ee03380547",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.44879546104892,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T09:21:00.000+0000",
      "latitude": 58.27780602585858,
      "estimatedRestoration": "2022-05-13T14:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT1047'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT1047",
      "reference": "JT1047",
      "engineerOnSiteTime": "2022-05-13T10:50:00.000+0000",
      "name": "HS2 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "HS2 9AU",
        "HS2 9AX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -6.761187569199687,
          58.27780602585858
        ],
        "type": "Point"
      },
      "UUID": "b3bced98-3b0c-46a5-99fc-c81922978347",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.761187569199687,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-13T14:06:00.000+0000",
      "latitude": 57.64147647384415,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT1246'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT1246",
      "reference": "JT1246",
      "name": "AB45 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "AB45 2AP",
        "AB45 2AQ",
        "AB45 2YQ",
        "AB45 2YR",
        "AB45 2YS"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.681963023115733,
          57.64147647384415
        ],
        "type": "Point"
      },
      "UUID": "2c9d8122-dbc6-44f9-9905-585ebe2c956b",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.681963023115733,
      "resolved": false
    }
  ]
}
